lld/test/ELF/linkerscript/locationcountererr.test

```diff
@@ -1,7 +1,7 @@
 # REQUIRES: x86
 # RUN: llvm-mc -filetype=obj -triple=x86_64-unknown-linux /dev/null -o %t
 # RUN: not ld.lld %t --script %s -o /dev/null 2>&1 | FileCheck %s
-# CHECK: {{.*}}.test:8: unable to move location counter backward for: .text
+# CHECK: {{.*}}.test:8: unable to move location counter (0x2000) backward (0x10) for section '.text'
```

Comment:
Maybe worth adding a test with more than one backwards error? Previously we'd get multiple reports now just the first one. There probably isn't much value in reporting further errors, but I think it does make sense to document this behaviour in a test?

Reply:
Thanks for the suggestion. 54daea8e15f7e5e95e6531d34616062af81c4d84 added two more memory regions to demonstrate the full set of errors.